### ISSUE TITLE
{-# LANGUAGE InstanceSigs #-}  shows red line error

### ISSUE THREAD

**Issue by ciez** [carymrobbins/intellij-haskforce#175]
with  InstanceSigs enabled, adding a signature to e.g. fmap shows an error.

GHC builds

**Comment by carymrobbins** [comment]
Test case below -

```haskell
data T a = MkT a a
instance Eq a => Eq (T a) where
  (==) :: T a -> T a -> Bool   -- The signature
  (==) (MkT x1 x2) (MkTy y1 y2) = x1==y1 && x2==y2
```

**Comment by carymrobbins** [comment]
This should be resolved in **v0.3-beta.22**.
